lib/shared/utils.js

```diff
@@ -124,6 +125,32 @@ function runScript(command, args) {
     });
 }
 
+function getCacheFiles(cacheDir) {
+    if (!fs.existsSync(cacheDir)) {
+        return {};
+    }
+
+    const cacheFilesBySlugs = fs.readdirSync(cacheDir).reduce((res, file) => {
+        const [, slug, timestamp] = file.match(/^\.discoveryjs\.(.*?)\.(\d+)\.cache$/);
+
+        if (slug && timestamp) {
+            res[slug] = res[slug] || [];
+            res[slug].push({
+                file,
```

Comment:
I think it better to return absolute path to file here

Reply:
Hmm, I think it'd be better if leave it untouched. Because there are many places where we using this path with other dirs.